Example/Tom/ViewController.swift

```diff
@@ -6,19 +6,99 @@
 //  Copyright (c) 2021 demirciy. All rights reserved.
 //
 
+import Tom
 import UIKit
 
 class ViewController: UIViewController {
 
+    // MARK: Properties
+
+    private lazy var tomView = TomView(configuration: .init(lineCount: 20, lineColor: retroCartoonsBlack))
+    private lazy var actionButtonStackView: UIStackView = {
+        let view: UIStackView = .init(arrangedSubviews: [startButton, stopButton])
+        view.axis = .horizontal
+        view.spacing = 32
+        view.distribution = .fillEqually
+        return view
+    }()
+    private lazy var startButton: UIButton = {
+        let button: UIButton = .init()
+        button.backgroundColor = retroCartoonsBlack
+        button.setTitleColor(retroCartoonsYellow, for: .normal)
+        button.setTitle("Start", for: .normal)
+        button.addTarget(self, action: #selector(startAction), for: .touchUpInside)
+        return button
+    }()
+    private lazy var stopButton: UIButton = {
+        let button: UIButton = .init()
+        button.backgroundColor = retroCartoonsBlack
+        button.setTitleColor(retroCartoonsYellow, for: .normal)
+        button.setTitle("Stop", for: .normal)
+        button.addTarget(self, action: #selector(stopAction), for: .touchUpInside)
+        return button
+    }()
+
+    private let retroCartoonsYellow: UIColor = .init(red: 228 / 255, green: 186 / 255, blue: 94 / 255, alpha: 1)
```

Comment:
```suggestion
    private let retroCartoonsYellow: UIColor = .init(red: 228 / 255, 
    green: 186 / 255, 
    blue: 94 / 255, 
    alpha: 1)
```

Reply:
We decided line length 120 therefore no need to send the lines below.